templates/kpi/config.es6.j2

```diff
@@ -6,4 +6,8 @@ export function getAuthUrl(url) {
 
 export function getZebraLoginUrl() {
   return KPI_LOGIN_URL;
-}
\ No newline at end of file
+}
+
+const SENTY_DSN = "{{ kpi_raven_js_dsn }}";
+
+export const ONA_TITLE = {{ kpi_ona_title }};
```

Comment:
Same as above, can we rename the `ONA_TITLE` to something like `WEB_PAGE_TITLE`?